spec.md

```diff
@@ -131,9 +141,9 @@ In order to test a registry's conformance against these workflow categories, ple
 
 #### Pull
 
-The process of pulling an artifact centers around retrieving two components: the manifest and one or more blobs.
+The process of pulling an Object centers around retrieving two components: the manifest and one or more blobs.
```

Comment:
seems odd to see "Object" capitalized

Reply:
i get it that it refers to https://github.com/opencontainers/distribution-spec/pull/335/files#diff-bc6661da34ecae62fbe724bb93fd69b91a7f81143f2683a81163231de7e3b545R74, but without wikipedia-level linking, it seems odd